### DISCUSSION TITLE
how to update remote config values from your Cloud Function in python?

### DISCUSSION THREAD

**Discussion by irfan767** [firebase/functions-samples#951] [Q&A]
Hi, I have some values in my remote config, And I fetched data from firestore in the cloud function. Now I want to update the data in the Remote Config from Cloud function but I cannot find the code in python. It's in Node JS.

**Comment by puf** [comment]
Hey irfan767. Good to hear that you're looking to use Remote Config in your app. 👍

This repo is a collection of samples for Cloud Functions, and discussion should focus around those samples.

If you'd like to ask for help on a coding problem, I recommend posting it to Stack Overflow with the relevant tags.

**Reply by irfan767** [reply]
Thank you so much for the reply. Please one thing I want to mention I am looking for a Cloud Function which is written in python that will update the data in Remote Config.  if you still don't understand this please read this. https://stackoverflow.com/questions/70540609/how-to-the-manipulated-data-from-google-cloud-function-to-the-firebase-remote-co?noredirect=1#comment124701579_70540609
